core/src/main/java/io/kestra/core/models/namespaces/NamespaceInterface.java

```diff
@@ -0,0 +1,13 @@
+package io.kestra.core.models.namespaces;
+
+import io.kestra.core.models.flows.PluginDefault;
+import io.kestra.core.models.tasks.WorkerGroup;
+
+import java.util.List;
+import java.util.Map;
+
+public interface NamespaceInterface {
```

Comment:
The interface can directly extend the `DeletedInterface` so that you don't need to redefine the `isDeleted` method.